projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { ReactElement } from 'react'
+import React, { ReactElement, useEffect } from 'react'
```

Comment:
think you left this around, remove if you can but feel free to keep it, it'll go away next time somebody sorts the imports!

Reply:
Fixed 👏